#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("enabled", bool_t, 0, "Whether to apply this plugin or not", True)
gen.add("lethal_radius", double_t, 0, "How much bigger mark by received point.", 1.0,1.0,5.0)

exit(gen.generate("cctv_layer", "cctv_layer", "CctvLayer"))
